pagopa/io-app
# feat(Bonus Pagamenti Digitali): [#175263900] Add winning transaction API

## Short description
This PR does some refactoring about spec naming and it adds the support for winning transactions API

### how to test

- app: `yarn generate:all`
- io-dev-server: make sure you have the last commit from `support-bpd branch` then `yarn generate:all; yarn start`
